test_app/schema.py

```diff
@@ -24,7 +24,7 @@
 class Episode(Object):
     name = CharField()
     number = IntegerField()
-    characters = ManyRelatedField(lambda: Character, types=List(String))
+    characters = ManyRelatedField('test_app.schema.Character', types=List(String))
```

Comment:
Note that this is also testing the new string import support and applies to all tests that uses `test_app/schema.py`.